cmd/discover.go

```diff
@@ -25,9 +25,9 @@ var discoverCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(discoverCmd)
-	discoverCmd.Flags().StringVar(&discoverOptions.GRPC, "gRPC", "", "gRPC server information")
+	discoverCmd.Flags().StringVar(&discoverOptions.GRPC, "grpc", "", "gRPC server information")
 	discoverCmd.Flags().StringVarP(&discoverOptions.Format, "format", "f", "json", "Format: json or yaml")
-	discoverCmd.Flags().StringVarP(&discoverOptions.Policy, "policy", "p", "kubearmor", "Type of policies to be discovered: cilium or kubearmor")
+	discoverCmd.Flags().StringVar(&discoverOptions.Policy, "class", "kubearmor", "Type of policies to be discovered: cilium or kubearmor")
```

Comment:
The class should be same in discover/insight.

Reply:
@nyrahul discover/insight or discover/discover ?

Reply:
We will need to have a consistent naming options for discover and insight. Please update cmd/discover.go to have application and network as options and application as default.

Reply:
Okay, so you will need to remove the cilium if condition and update kubearmor condition with `o.Policy == "application"` instead

Reply:
@Ankurk99 re-review